sdk/js/src/utils/consts.ts

```diff
@@ -78,6 +80,10 @@ export const CosmWasmChainNames = [
   "injective",
   "xpla",
   "sei",
+  "wormchain",
```

Comment:
should we call this `gateway` instead of `wormchain`?

Reply:
Changed